lib/application/widgets/animatable_progress.dart

```diff
@@ -115,11 +115,11 @@ class AnimatableLinearProgress extends AnimatableProgress implements ProgressPai
   final double? minWidth;
 
   @override
-  _AnimatableLinearProgressState createState() => _AnimatableLinearProgressState();
+  AnimatableLinearProgressState createState() => AnimatableLinearProgressState();
```

Comment:
You don't need to expose the state, you just have to change the return type to
```suggestion
  AnimatableProgressState<AnimatableLinearProgress> createState() => AnimatableLinearProgressState();
```